components/Currency/CurrencyPrice/CurrencyPrice.tsx

```diff
@@ -59,10 +57,15 @@ export const CurrencyPrice: FC<CurrencyPriceProps> = memo(
 	}
 );
 
-const Container = styled.span`
+const Container = styled.span<{ $side?: 'positive' | 'negative' }>`
 	${ContainerRowMixin};
 	font-family: ${(props) => props.theme.fonts.mono};
 	color: ${(props) => props.theme.colors.selectedTheme.button.text.primary};
+	${(props) =>
+		!!props.$side &&
+		css`
+			color: ${props.theme.colors.selectedTheme.newTheme.text.number[props.$side]};
```

Comment:
Not sure this is working as expected, see the comment on `FuturesMarketsTable` and the open interest colors

Reply:
Yeah. The issue was I wasn't passing the `$side` prop from the parent component, but I've pushed a fix now.